pros/serial/ports/v5_wireless_port.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, port):
         self.download_channel.__enter__()
 
     def destroy(self):
+        self.port_instance.destroy()
         self.download_channel.__exit__()

```

Comment:
This writes to the port_instance after it's been destroyed. @theol0403 Do you know why it was failing when port_instance was destroyed after this.

Reply:
Or rephrased, why is __exit__ causing this problem?

Reply:
the interrupt handler for the terminal command calls down to this function. i'm not sure what you're asking?

Reply:
What I think is happening is that port_instance destroys the port_instance, then the call to exit just errors and returns.

Reply:
the second call is not acting on the port_instance?

Reply:
It stores it in the class when it's constructed.

Reply:
well that doesn't destroy the port either, which is by design. also technically the download channel is given a device made from the port. granted, yes, this call should go after the switch back to pit channel.... however @theol0403 mentioned this fix didn't work unless the call to destroy took place first.

i landed on this as a possible fix after looking at the program flow:
terminal.stop -> device.destroy, where device is either a direct port or a wireless port. the direct port's destroy method then closes the serial port, while the wireless port's just calls for a switch back to pit channel, where we might expect it to delegate to its internal direct port's destroy method.

idk if any of that makes sense, but i hope you can follow my train of thought at least somewhat. plus, experimentally this has seemed to work...